src/store/modules/auth.store.js

```diff
@@ -22,8 +22,7 @@ export const authStore = {
             userService.setting(profilePicture, username, bio, password1, password2)
             .then(
                 user => {
-                    // localStorage.setItem('user', JSON.stringify(data));
-                    localStorage.setItem('user', username);
+                    localStorage.setItem('user', JSON.stringify(data));
```

Comment:
```
storedProfile = JSON.parse(localStorage.getItem('user'));
storedProfile.profile = user;
localStorage.setItem('user', JSON.stringify(storedProfile));
```

Reply:
You can also `console.log(user)` to check if you are getting expected response as you saw in Postman.